builder/amazon/common/step_run_source_instance.go

```diff
@@ -84,24 +84,18 @@ func (s *StepRunSourceInstance) Run(state multistep.StateBag) multistep.StepActi
 	}
 
 	ui.Say("Launching a source AWS instance...")
-	imageResp, err := ec2conn.DescribeImages(&ec2.DescribeImagesInput{
```

Comment:
Previously we made the `DescribeImages` call twice, rather than using our state bag. This now uses the result in the state bag.